internal/backends/postgresql/metadata/pool/opendb.go

```diff
@@ -52,12 +52,13 @@ func openDB(uri string, l *zap.Logger, sp *state.Provider) (*pgxpool.Pool, error
 		ctx, cancel := context.WithTimeout(ctx, 2*time.Second)
 		defer cancel()
 
-		if err = conn.QueryRow(ctx, `SHOW server_version`).Scan(&v); err != nil {
+		if err = conn.QueryRow(ctx, `SELECT version()`).Scan(&v); err != nil {
 			return lazyerrors.Error(err)
 		}
 
-		if sp.Get().HandlerVersion != v {
-			if err = sp.Update(func(s *state.State) { s.HandlerVersion = v }); err != nil {
+		n, v, _ := strings.Cut(v, " ")
```

Comment:
I see, so `v` includes version and additional system info like
`16.0 (Debian 16.0-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14)`

Reply:
And `n` contains `PostgreSQL` or [`CockroachDB`](https://stackoverflow.com/a/73324156) or something